pysb/integrate.py

```diff
@@ -145,7 +217,14 @@ def rhs(t, y, p):
         else:
             self.yexpr = numpy.ndarray((len(tspan), 0))
         self.yobs_view = self.yobs.view(float).reshape(len(self.yobs), -1)
-        self.integrator = ode(rhs).set_integrator(integrator, **options)
+        # Initialize an instance of scipy.integrate.ode
+        if self._use_jacobian:
```

Comment:
In conditional, just set value of local "jac" variable, then pass that in a single call to ode(). You'll need to figure out the default value of jac (possible None?) in the ode definition.